presto-main/src/main/java/com/facebook/presto/cost/PlanNodeCostEstimate.java

```diff
@@ -27,6 +27,21 @@
     public static final PlanNodeCostEstimate UNKNOWN_COST = new PlanNodeCostEstimate(NaN, NaN, NaN);
     public static final PlanNodeCostEstimate ZERO_COST = new PlanNodeCostEstimate(0, 0, 0);
 
+    public static PlanNodeCostEstimate individualCpuCost(double individualCpuCost)
+    {
+        return new PlanNodeCostEstimate(individualCpuCost, 0, 0);
+    }
+
+    public static PlanNodeCostEstimate individualMemoryCost(double individualMemoryCost)
+    {
+        return new PlanNodeCostEstimate(0, individualMemoryCost, 0);
+    }
+
+    public static PlanNodeCostEstimate individualNetworkCost(double individualNetworkCost)
+    {
+        return new PlanNodeCostEstimate(0, 0, individualNetworkCost);
+    }
+
     public static PlanNodeCostEstimate cpuCost(double cpuCost)
```

Comment:
where do these get used.  Right now they are exactly the same as the individualCost methods.  Figure out if that's the correct behavior, and if so get rid of one set of methods.